src/index.html

```diff
@@ -11,7 +11,166 @@
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="block">
+          Full date of birth:
+          <input
+            class="form__inp"
+            type="date"
+            name="birthday"
+          >
+        </label>
+        <label class="block">
+          I accept the term of the agreement
+          <input
+            class="form__inp"
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__grp">
+        <legend>Registration</legend>
+        <label class="block">
+          E-mail:
+          <input
+            class="form__inp"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            class="form__inp"
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="10"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__grp">
+        <legend>An interesting fact about you!</legend>
+        <div class="block">
+          Do you love cats?
+          <label>
+            <input
+              class="form__inp"
+              type="radio"
+              name="lovecats"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              class="form__inp"
+              type="radio"
+              name="lovecats"
+            >
+            No
+          </label>
+        </div>
+        <label class="block">
+          What is your favorite color?
+          <input
+            class="form__inp"
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="block">
+          What time do you go to bed?
+          <input
+            class="form__inp"
+            type="time"
+            name="time"
+          >
+        </label>
+        <label class="form__inp block">
+          What are your favourite brand of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="block">
+          How do you rate our work?
+          <input
+            class="form__inp"
+            type="range"
+            name="rate"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__grp">
+        <legend>Additional info:</legend>
+        <label class="form__inp block">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="331" alt="image" src="https://user-images.githubusercontent.com/53611910/205504765-afaef63e-4f8c-48ba-b109-3dd1e70a1769.png">

Reply:
"></textarea>" 
this way I get an error:
151:11  error  Tag’s closing bracket should be on the same level 
as opening one  (closing-and-opening-on-seme-level)
![closing_tag](https://user-images.githubusercontent.com/91250208/205514635-54a42631-5748-4c92-a3bc-8e0e2c065092.png)